private/bufpkg/bufcheck/buflint/testdata/validate_rules_types_match_success/buf.yaml

```diff
@@ -0,0 +1,7 @@
+version: v1
+lint:
+  use:
+    - VALIDATE_CONSTRAINTS_CHECK
+  ignore:
```

Comment:
Nit: optimally, `testLintWithModifiers` would automatically add an `ignore` for the dependency directory it added, which can be done via `configModifier`.